README.md

```diff
@@ -1,7 +1,19 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
 
+# The databases can be found using the local databases as follows:
+PGDATABASE=nc_news;
+PGDATABASE=nc_news_test;
+
```

Comment:
I don't think this is very clear as to what someone would have to do in order to run your server:

I'd tweak this to explicitly mention the creation of .env files and what they'll have to be called, to avoid any potential confusion